docs/release_notes/release_notes-template.rst

```diff
@@ -27,7 +27,7 @@ Key Features and Updates
   *
 * Dependencies
   *
+* New Features
 
 Contributors
 ------------
-
```

Comment:
why was this one removed?

Reply:
That's a typo! Fixed

Reply:
thats odd - vscode shows an additional newline at the end of the file in my version. Let me try adding an extra extra newline and see what happens.